src/command/graph/push.rs

```diff
@@ -54,15 +63,15 @@ impl Push {
             },
             &client,
         )
-        .context("Failed while pushing to Apollo Studio")?;
+        .context("Failed while pushing to Apollo Studio. To see a full printout of the schema attempting to push, rerun with `--log debug`")?;
```

Comment:
would be awesome if we could try to figure out _what_ went wrong and what steps can be taken to remediate them by matching on the different error types.

Reply:
I feel like this is a _little_ out of scope for this PR but opened an issue #111 so we don't forget :)